app/components/blacklight/response/spellcheck_component.rb

```diff
@@ -8,7 +8,12 @@ class SpellcheckComponent < ViewComponent::Base
       # @param [Array<String>] options explicit spellcheck options to render
       def initialize(response:, options: nil)
         @response = response
-        @options = options || @response&.spelling&.words
+        @options = options
```

Comment:
There's a change in behavior here that I don't think is intentional: Prior to the PR, a passed options param will always provide the value of `@options`; here, either form of spelling suggestion will override it. Was this intentional?